test/tools/runner/hooks/configuration.js

```diff
@@ -149,7 +149,8 @@ const testConfigBeforeHook = async function () {
     ldap: MONGODB_URI.includes('authMechanism=PLAIN'),
     ocsp: process.env.OCSP_TLS_SHOULD_SUCCEED != null && process.env.CA_FILE != null,
     socks5: MONGODB_URI.includes('proxyHost='),
-    compressor: process.env.COMPRESSOR
+    compressor: process.env.COMPRESSOR,
+    cryptSharedLibPath: process.env.CRYPT_SHARED_LIB_PATH
```

Comment:
Set to empty string: https://evergreen.mongodb.com/task_log_raw/mongo_node_driver_next_ubuntu1804_custom_dependency_tests_run_custom_csfle_shared_lib_tests_pinned_commit_patch_c70c69edb7b51aeac143aa2eb73aaaf465c41572_62f13ed0562343194bc2de30_22_08_08_16_50_25/0?type=T#L801